django_spanner/introspection.py

```diff
@@ -303,14 +295,13 @@ def get_constraints(self, cursor, table_name):
             RIGHT JOIN
                 INFORMATION_SCHEMA.INDEX_COLUMNS AS idx_col
             ON
-                idx_col.INDEX_NAME = idx.INDEX_NAME AND idx_col.TABLE_NAME="{table}"
+                idx_col.INDEX_NAME = idx.INDEX_NAME AND idx_col.TABLE_NAME=@table
```

Comment:
This type of join clauses should also include the schema name. Otherwise, two indexes (or tables) that have the same name, but live in different schemas, could be joined here.

```suggestion
                idx_col.INDEX_NAME = idx.INDEX_NAME AND idx_col.TABLE_NAME=@table AND idx_col.TABLE_SCHEMA=idx.TABLE_SCHEMA
```